.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,88 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image_for_test: "ibmcom/ibm-block-csi-operator:latest"
+
+  prepare_env:
+   name: "prepare env"
+   needs: operator_image_build
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup dependencies
+        run: |
+          build/ci/community/setup_ci_dependencies.sh
+      - name: Save dependencies files
+        uses: actions/upload-artifact@v2
+        with:
+          name: gitconfig-file
+          path: |
+            /home/runner/.gitconfig
+            /home/runner/.bash_profile
+          retention-days: 1
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Prepare env
+        id: environment_setup
+        run: |
+          build/ci/community/prepare_env.sh
+   outputs:
+     community_operators_path: "${{ steps.environment_setup.outputs.community_operators_path }}"
+     upstream_community_operators_path: "${{ steps.environment_setup.outputs.upstream_community_operators_path }}"
+     repository_path: "${{ steps.environment_setup.outputs.repository_path }}"
+     operator_image_for_test: ${{ needs.operator_image_build.outputs.operator_image_for_test }}
+     csv_file:  ${{ steps.environment_setup.outputs.csv_file }}
+
+  CI-csi-demo-pr:
+    name: "community-operators fork checks"
+    needs: prepare_env
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    env:
+      original_community_operators_repository: "k8s-operatorhub/community-operators"
+      original_community_operators_repository_prod: "redhat-openshift-ecosystem/community-operators-prod"
+      forked_community_operators_repository: "csiblock/community-operators"
```

Comment:
please [change the `forked` terminology to `fork`](https://github.com/IBM/ibm-block-csi-operator/pull/157#discussion_r687627546) wherever applicable